stubs/openpyxl/openpyxl/cell/cell.pyi

```diff
@@ -1,42 +1,52 @@
-from _typeshed import Incomplete
-from datetime import datetime
+import datetime
+from _typeshed import ReadableBuffer
+from re import Pattern
+from typing_extensions import Final, Literal, TypeAlias
 
 from openpyxl.comments.comments import Comment
+from openpyxl.compat.numbers import NUMERIC_TYPES as NUMERIC_TYPES, _NumericTypes
 from openpyxl.styles.cell_style import StyleArray
 from openpyxl.styles.styleable import StyleableObject
+from openpyxl.workbook.child import _WorkbookChild
 from openpyxl.worksheet.hyperlink import Hyperlink
-from openpyxl.worksheet.worksheet import Worksheet
 
 __docformat__: str
-TIME_TYPES: Incomplete
-TIME_FORMATS: Incomplete
-STRING_TYPES: Incomplete
-KNOWN_TYPES: Incomplete
-ILLEGAL_CHARACTERS_RE: Incomplete
-ERROR_CODES: Incomplete
-TYPE_STRING: str
-TYPE_FORMULA: str
-TYPE_NUMERIC: str
-TYPE_BOOL: str
-TYPE_NULL: str
-TYPE_INLINE: str
-TYPE_ERROR: str
-TYPE_FORMULA_CACHE_STRING: str
-VALID_TYPES: Incomplete
+TIME_TYPES: tuple[type[_TimeTypes], ...]
+_TimeTypes: TypeAlias = datetime.datetime | datetime.date | datetime.time | datetime.timedelta
+TIME_FORMATS: dict[_TimeTypes, str]
+STRING_TYPES: tuple[type[str], type[bytes]]
+_StringTypes: TypeAlias = str | bytes
+# We could get the exact tuple length and order with the following line, ignoring Y015
+# But pytype doesn't support it and mypy oversimplifies the type. Pyright fully expands it.
+# KNOWN_TYPES = NUMERIC_TYPES + TIME_TYPES + STRING_TYPES + (bool, type(None))
```

Comment:
I feel like we don't really need this comment -- stubs are meant to be static declarations of types, after all. It's kind of understood that we want to keep dynamic inference of types to a minimum, because that's sort of the whole point of stubs :D

```suggestion
```

Reply:
This comment should've gonna away with your last batch of suggestions anyway. Definitely no longer relevant!